common/persistence/executionStore.go

```diff
@@ -78,20 +78,17 @@ func (m *executionManagerImpl) GetWorkflowExecution(
 	}
 	newResponse := &GetWorkflowExecutionResponse{
 		State: &WorkflowMutableState{
-			ActivityInfos:      response.State.ActivityInfos,
-			TimerInfos:         response.State.TimerInfos,
-			RequestCancelInfos: response.State.RequestCancelInfos,
-			SignalInfos:        response.State.SignalInfos,
-			SignalRequestedIDs: response.State.SignalRequestedIDs,
-			ReplicationState:   response.State.ReplicationState,
-			Checksum:           response.State.Checksum,
+			ActivityInfos:       response.State.ActivityInfos,
+			TimerInfos:          response.State.TimerInfos,
+			RequestCancelInfos:  response.State.RequestCancelInfos,
+			SignalInfos:         response.State.SignalInfos,
+			SignalRequestedIDs:  response.State.SignalRequestedIDs,
+			ReplicationState:    response.State.ReplicationState,
+			Checksum:            response.State.Checksum,
+			ChildExecutionInfos: response.State.ChildExecutionInfos,
```

Comment:
`ChildExecutionInfos` wasn't there at all?

Reply:
See the part I removed below.